### PR TITLE
add new event specs

### PR DESCRIPTION
**Pull request by kylestengline**
Adds testing for the new event functionality. Let me know if you think the flash messages or process should be different and I can add accordingly. 
Cheers.
Referencing issue: [#8 ](https://github.com/dev-magic/letsmeetup/issues/8)

### CODE REVIEW

File: app/models/preference.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Expresses someone's availability for a specific day/time
 class Preference < ApplicationRecord
-  validates :preference_type, presence: true
+  validates_presence_of :preference_type
```
Comment (appleJax) [inline app/models/preference.rb]: `validates_presence_of` is the old version, right? Shouldn't we be using `validates :preference_type, presence: true` ?

File: .hound.yml [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
   config_file: config/.jshintrc
 
 ruby:
-  config_file: .rubocop.yml
\ No newline at end of file
+  config_file: .rubocop.yml
+=======
+ruby:
+  config_file: .rubocop.yml
+>>>>>>> pass model specs, add and configure database_cleaner, validate models
```
Comment (appleJax) [inline .hound.yml]: Unresolved merge conflict

File: app/models/timeslot.rb [hunk 3]
```diff
@@ -3,5 +3,5 @@ class Timeslot < ApplicationRecord
   validates :start_time, presence: true
 
   belongs_to :event
-  has_many :preferences, through: :event, class_name: 'Event'
+  has_many :preferences, through: :event, class_name: "Event"
```
Comment (appleJax) [inline app/models/timeslot.rb]: I thought we were preferring single quotes when double quotes are not necessary

File: app/models/user.rb [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # An event participant
 class User < ApplicationRecord
-  validates :name, presence: true
-  validates :email, presence: true
+  validates_presence_of :name, :email
```
Comment (appleJax) [inline app/models/user.rb]: `validates_presence_of` is the older version, no?

File: app/models/timeslot.rb (hunk 3 above)
Comment (kylestengline) [reply app/models/timeslot.rb]: We are, but I changed this when I made that first push. I'm not sure how it got back in there.

File: app/models/user.rb (hunk 4 above)
Comment (kylestengline) [reply app/models/user.rb]: You're right, but when I made my first pull request, this was changed to the `validates` instead of `validates_presence_of`.

Comment (kylestengline) [reply app/models/user.rb]: You're right, but when I made my first pull request, this was changed to the `validates` instead of `validates_presence_of`.

**Comment by appleJax** [comment]
Looks good to me! 🎉

**Pull request merged**